frontend/src/components/App/Home/index.tsx

```diff
@@ -209,6 +209,17 @@ function useWarningSettingsPerCluster(clusterNames: string[]) {
   return warningLabels;
 }
 
+function getOrigin(cluster: Cluster) {
```

Comment:
Please add function documentation?

Reply:
Maybe the return type could be added `function getOrigin(cluster: Cluster): string {` instead of in the returns.

`@returns the description of where the cluster is picked up from. Dynamic, in-cluster or from a kubeconfig file`

Reply:
done